tests/stores/test_agent_tokens_store.py

```
@@ -0,0 +1,105 @@
+
+
+@pytest.mark.asyncio
+async def test_issue_stores_hash_not_plaintext(store):
+    plaintext, row = await store.issue(agent_id="a", user_id="u", scope=["*"])
+    assert row["token_hash"] != plaintext
+    assert len(row["token_hash"]) == 64
+
+
+@pytest.mark.asyncio
+async def test_issue_revokes_previous_token_atomically(store):
+    plaintext_a, row_a = await store.issue(agent_id="agent-1", user_id="u", scope=["*"])
```

Ruff (0.15.12)

[warning] Unpacked variable `row_a` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)